exe/Main.hs

```diff
@@ -28,11 +28,13 @@ import Path.IO qualified as Path
 import Yare.App qualified as Yare
 import Yare.App.Types qualified as App
 import Yare.Chain.Point (ChainPoint, parseChainPoint)
+import Yare.Config.Toml qualified as Toml
```

Comment:
Its better not to use `TOML` in the name as it is an module's implementation detail rather than a role:
we need to think of what is this module responsible for but not how it achieves it. This way we can hypothetically change toml to yaml without renaming the module. 

I recognize that name `Config` is the most appropriate one and its already used. So this is actually a hint that we might not need 2 different configs, or at least we need a rule of thumb to decide where to put a parameter.

Reply:
That's a fair point!